src/_components/CentreModal/CentreModal.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createOfficeLocation } from '../../_actions/officeLocations.actions';
+
+import CentreModalComponent from '../../components/CentreModal/CentreModal';
+
+class CentreModal extends Component {
+  state = {
+    name: '',
+    country: ''
+  };
+
+  onAddCentre = (event) => {
+    this.setState({ name: event.target.value });
+  };
+
+  onAddCountry = (event) => {
```

Comment:
Give both `country` and`name` fields appropriate names, then combine `onAddCentre` and `onAddCountry` into a single method such as `handleChange`. Read more about how to handle multiple form inputs [here](https://reactjs.org/docs/forms.html#handling-multiple-inputs)